dashboard/src/components/skills/catalog/ImportedSkillInfo.vue

```diff
@@ -31,14 +31,27 @@ limitations under the License.
         </template>
 
         <template v-slot:cell(importingProjectName)="data">
-          <span class="ml-2">{{ data.value }}</span>
-          <span v-if="data.item.enabled !== 'true'" class="text-uppercase ml-2"><b-badge variant="warning">Disabled</b-badge></span>
+          <div class="row">
+            <div class="col">
+              <span class="ml-2">{{ data.value }}</span>
+              <span v-if="data.item.enabled !== 'true'" class="text-uppercase ml-2"><b-badge variant="warning">Disabled</b-badge></span>
+            </div>
+            <div class="col-auto">
+              <b-button variant="outline-primary" :aria-label="`Contact ${data.item.name} project owner`"
+                               @click="chooseProject(data.value, skill.projectId)" :data-cy="`contactOwnerBtn_${ skill.projectId }`">
```

Comment:
looks like the `project id` method paramter to the `chooseProject` method is incorrect and will be the current project (where the skill is exported); it should be the `project id` for where the skill is imported (`project id` for the current table row); looks like an email will be sent to the wrong project